pkg/utils/mco_deploy.go

```diff
@@ -418,7 +418,6 @@ func ModifyMCOCR(opt TestOptions) error {
 	spec := mco.Object["spec"].(map[string]interface{})
 	spec["retentionResolutionRaw"] = "3d"
 	spec["nodeSelector"] = map[string]string{"kubernetes.io/os": "linux"}
-	spec["availabilityConfig"] = "Basic"
```

Comment:
I do not understand why we delete this line?

Reply:
I found that this case can be failure in most case

Reply:
you mean we do not need this case? maybe we can add retry func to handle it.

Reply:
that is not a P1 case for release-2.2. You may know that it spends more time to switch from Basic to High and back. I think we can just disable it to reduce the failure.